cla_public/apps/checker/tests/test_validation.py

```diff
@@ -63,7 +63,10 @@ def test_email_validator(self):
         field.data = 'emailwithspace@example.com '
         self.assertRaises(ValidationError, validator, form, field)
 
-        field.data = 'emailwithspace@example.com'
+        field.data = 'emailwith,comma@example.com'
+        self.assertRaises(ValidationError, validator, form, field)
+
+        field.data = 'email.with-space@example.com'
```

Comment:
yes, the fixture should be `field.data = 'email.with-hyphen@example.com'` instead of `'email.with-space@example.com'`